docroot/profiles/custom/cgov_site/modules/custom/pdq_cancer_information_summary/pdq_cancer_information_summary.module

```diff
@@ -191,7 +192,7 @@ function pdq_cancer_information_summary_node_view(array &$build, EntityInterface
  * @param string $url
  *   The optional value to match for the pretty url field.
  *
- * @return Drupal\taxonomy\Entity\Term
+ * @return mixed
```

Comment:
This should probably be like
`@return \Drupal\taxonomy\Entity\Term|false`

Reply:
Resolved.